src/sentry/event_manager.py

```diff
@@ -510,6 +509,9 @@ def save(self, project, raw=False):
         else:
             transaction_name = culprit
 
+        if not isinstance(culprit, six.string_types):
```

Comment:
There’s no need to check the type, `force_text` will just noop if the input is already unicode.